Biasing/include/Biasing/TargetDarkBremFilter.h

```diff
@@ -83,7 +83,7 @@ class TargetDarkBremFilter : public simcore::UserAction {
    *
    * @param[in] step current G4Step
    */
-  void stepping(const G4Step* step);
+  void stepping(const G4Step* step) final override;
```

Comment:
Having both override and final is redundant since final implies override (i would opt for override unless we have a specific reason for forbidding subtyping)

Reply:
Oh ok... I just saw that we have already (I mean before this PR) 86 files with `final override` 
https://github.com/search?q=repo%3ALDMX-Software%2Fldmx-sw%20final%20override&type=code
so I'd have it consistent, should I change all of those too to just `override`?

Reply:
Hmm, looking into this quickly in stackoverflow, people seem to be ok with having `final override`